designer/client/helpers.js

```diff
@@ -68,3 +70,11 @@ export function camelCase(str) {
     .replace(/[\s-_]+(.)/g, (m, chr) => chr.toUpperCase())
     .replace(/[^a-zA-Z0-9]/g, "");
 }
+
+export function str2bool(value) {
+  if (value && typeof value === "string") {
+    if (value.toLowerCase() === "true") return true;
+    if (value.toLowerCase() === "false") return false;
+  }
+  return value;
+}
```

Comment:
see comment above. Also, this code probably doesn't do what you're expecting it to. I would assume str2bool will return a boolean, but it can return a string if it is not 'true' or 'false'